.aws/task-definition.json

```diff
@@ -0,0 +1,61 @@
+{
+  "containerDefinitions": [
+    {
+      "name": "tis-trainee-forms",
+      "image": "430723991443.dkr.ecr.eu-west-2.amazonaws.com/tis-trainee-forms:eceec14565f633ce0ccc4ad2cfd812bf3ff469d4",
```

Comment:
The commit ID at the end may need to be changed

Reply:
Which commit ID?

Reply:
'eceec14565f633ce0ccc4ad2cfd812bf3ff469d4' It is an ID for one the commit in trainee-details service (https://github.com/Health-Education-England/tis-trainee-details/commit/eceec14565f633ce0ccc4ad2cfd812bf3ff469d4)

Reply:
Done